### ISSUE TITLE
How to capture axios requests as a dependency metric? 

### ISSUE THREAD

**Issue by KarineValenca** [labbsr0x/express-monitor#23]
I do a request to an external service using an axios request. I want to collect metrics of this request, however, axios doesn't provide the variables needed for the collect dependency metric method:

```javascript
Monitor.collectDependencyTime(req, res, name, type)
``` 

How to capture axios requests as a dependency metric? It's possible using the current endpoint?

**Comment by KarineValenca** [comment]
After some tests, I realized that the way `collectDependencyTime` is implemented isn't generic enough to capture different kinds of dependencies. 
Right now, `collectDependencyTime` accepts `req` and `res` params. These arguments are `express.Request` and `express.Response`, respectively. So, it only allows collecting express dependencies, limiting the possibilities of collecting dependencies metrics.

**Comment by KarineValenca** [comment]
I changed the way the method is implemented in this PR #24 and added an example about how to use the method to collect axios metrics:

```javascript
app.get("/axios", async (req, res) => {
  const start = process.hrtime()
  try{
    //using a service to create a slow request
    const response = await axios.get('http://slowwly.robertomurray.co.uk/delay/500/url/http://google.com/')
    const { method, path } = response.request

    Monitor.collectDependencyTime("Google", "axios", response.status, method, path, "", start)
    res.json({"ok": true})
  }catch(err){
    if (err.request) {
      const { method, path } = err.request._options
      Monitor.collectDependencyTime("Google", "axios", 404, method, path, "endpoint not found", start)
      res.json({"ok": false})
    }else{
      Monitor.collectDependencyTime("Google", "axios", 500, "GET", "/err", "server error", start)
      res.json({"ok": false})
    }
  }
})
```
